### PR TITLE
Revert container security context

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+gen-expected:
+	helm3 dependency update tests/chart
```
Comment (danduk82) [inline Makefile:2]: helm3 or helm ? In the other charts it's just `helm`

**Comment by sbrunner** [comment]
@danduk82 UP ...

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+HELM != helm
```
Comment (danduk82) [inline Makefile:1]: +1

**Review by danduk82 — APPROVED** [review]
lgtm

**Review by danduk82 — APPROVED** [review]
lgtm

**Pull request merged**